src/MissingNumber.java

```diff
@@ -0,0 +1,28 @@
+public class MissingNumber {
+
+	void displayMissingNum(int[] intArr) {
+		
+		boolean flag;
+		System.out.println(" Missing Numbers are : ");
+		for (int i=1; i<=10; i++) {
```

Comment:
Please use index instead of i, it will be good during interview as well as in regular practice.